zookeeper-server/src/main/java/org/apache/zookeeper/cli/SyncCommand.java

```diff
@@ -55,14 +57,18 @@ public CliCommand parse(String[] cmdArgs) throws CliParseException {
     @Override
     public boolean exec() throws CliException {
         String path = args[1];
+        CountDownLatch latch = new CountDownLatch(1);
+        final int[] resultCode = new int[1];
```

Comment:
Why is an array necessary here? Wouldn't a plain integer be sufficient?

Reply:
when use a plain integer, a anonymous Inner Class needs it to be modified with `final `,then we cannot assignment and transfer the result outside.